proof/proofplayer/src/TProofDraw.cxx

```diff
@@ -779,15 +779,15 @@ void TProofDrawHist::SlaveBegin(TTree *tree)
       fTreeDrawArgsParser.Parse(fInitialExp, fSelection, fOption);
       fDimension = fTreeDrawArgsParser.GetDimension();
       TString exp = fTreeDrawArgsParser.GetExp();
-      const char *objname = fTreeDrawArgsParser.GetObjectName();
-      if (objname && strlen(objname) > 0 && strcmp(objname, "htemp")) {
-         TH1 *hist = dynamic_cast<TH1*> (fInput->FindObject(objname));
+      TString objname = fTreeDrawArgsParser.GetObjectName();
```

Comment:
Tehcnically this should work:
```suggestion
      const TString& objname = fTreeDrawArgsParser.GetObjectName();
```
but is indeed rely on the implementation of `GetObjectName` to not change.